window.c

```diff
@@ -119,10 +119,14 @@ void win_init(win_t *win)
 	f = win_res(db, RES_CLASS ".font", "monospace-8");
 	win_init_font(e, f);
 
-	bg = win_res(db, RES_CLASS ".background", "white");
-	fg = win_res(db, RES_CLASS ".foreground", "black");
-	win_alloc_color(e, bg, &win->bg);
-	win_alloc_color(e, fg, &win->fg);
+	bg_color = win_res(db, RES_CLASS ".background", "white");
+	fg_color = win_res(db, RES_CLASS ".foreground", "black");
+	bar_color = win_res(db, RES_CLASS ".bar", fg_color);
+	text_color = win_res(db, RES_CLASS ".text", bg_color);
```

Comment:
As mentioned by @eylles , `.bar.foreground` and `.bar.background` would be better.